### ISSUE TITLE
Swagger UI doesn't send cookie to backend

### ISSUE THREAD

**Issue by chikko80** [poem-web/poem#447]
I've defined a security schema that extracts a refresh token from the cookie.

```rust
#[derive(SecurityScheme)]
#[oai(
    type = "api_key",
    key_name = "refresh_token",
    in = "cookie",
    checker = "valid_refresh_token"
)]
pub struct UserWithValidRefreshToken(User);
```

The cookie is present in the browser and the request runs fine if u do the curl request and append the cookie value manually. However, if I try to trigger the endpoint via the SwaggerUI, the backend doesn't receive any cookie.
What am I missing here?

**Comment by TheCataliasTNT2k** [comment]
Important to be fixed...

**Comment by TheCataliasTNT2k** [comment]
It does not, because this is not really possible.
See link above:
> Note for Swagger UI and Swagger Editor users: Cookie authentication is currently not supported for "try it out" requests due to browser security restrictions. See [this issue](https://github.com/swagger-api/swagger-js/issues/1163) for more information. [SwaggerHub](https://swagger.io/tools/swaggerhub/) does not have this limitation.

You can use Postman, Insomnia and all of these instead.

**Comment by chikko80** [comment]
@TheCataliasTNT2k 

Mhm, do you think we should close this?

**Comment by TheCataliasTNT2k** [comment]
No. This can not "repair" the functionality within the browser right now, but the feature is still missing in poem. There is a possibility to add it to swagger, so it should be done. When it works within the browser, this feature will be helpful, but for documentary reasons it is definitely a good idea, to inplement it even before that.
